### PR TITLE
python3Packages.simpleldap: remove

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

Fairly outdated and incompatible with Python 3, see the following links
for further reference:

* https://github.com/gdub/python-simpleldap/blob/0.8/setup.py#L12-L14
* #43782 (comment)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dotlambda** [comment]
Judging from https://github.com/gdub/python-simpleldap/blob/0.8/setup.py#L12-L14, we should set `disabled = isPy3k`.

File: pkgs/development/python-modules/simpleldap/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage, ldap, pep8, pytest, tox, lib, fetchFromGitHub, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "simpleldap";
+  version = "0.8";
+
+  src = fetchFromGitHub {
```
Comment (dotlambda) [inline pkgs/development/python-modules/simpleldap/default.nix]: What's the reason you switched to `fetchFromGitHub`? Please add an inline comment.

**Comment by dotlambda** [comment]
I think we should just remove this package: It hasn't been updated since 2014 and its only dependent, inginious, is marked as broken since https://github.com/NixOS/nixpkgs/commit/611116823a13930caf2bf0402300b9b523efb92b. It should either be updated to the most recent version, which replaces `simpleldap` by `ldap3` or completely removed as @layus suggested in https://github.com/NixOS/nixpkgs/pull/31975#issuecomment-346647857.

**Comment by Ma27** [comment]
@dotlambda thanks for elaborating, I dropped both of these packages, a broken one is obviiously not in use and `simpleldap` is way too outdated which is why I should use a more recent alternative :sweat_smile:

**Comment by dotlambda** [comment]
> I dropped both of these packages

Please adjust the commit message of the first commit accordingly.

**Comment by Ma27** [comment]
argh, good catch, sorry :/

**Pull request merged**